demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -4070,8 +4070,16 @@ def test_passing_validation_using_git(self, mocker, repo):
         valid_script_yml = get_yaml(VALID_SCRIPT_PATH, cache_clear=True)
         pack2 = repo.create_pack("PackName2")
         script = pack2.create_script(yml=valid_script_yml)
+
         old_integration = pack2.create_integration(
-            "OldIntegration", yml={"toversion": "5.0.0", "deprecated": True}
+            name="OldIntegration", description="Deprecated. No available replacement."
+        )
+        old_integration.yml.update(
```

Comment:
you can use `|=` operator instead of update

Reply:
Tried to replace with:
```python
        old_integration.yml |= {
            "display": "OldIntegration (Deprecated)",
            "toversion": "5.0.0",
            "deprecated": True,
        }
```

I'm getting the following error:
```TypeError: unsupported operand type(s) for |=: 'YAML' and 'dict'```

Turns out `old_integration.yml` is a `YAML` object with a different `update` method than `dict`'s `update`, so `|=` doesn't work.